5.0/en/0x13-V5-Validation-Sanitization-Encoding.md

```diff
@@ -56,6 +56,7 @@ For example:
 | **5.2.9** | [ADDED] Verify that the application uses slashes to correctly escape special characters being used in regular expressions to ensure they are not misinterpreted as control characters. | ✓ | ✓ | ✓ | 624 |
 | **5.2.10** | [ADDED] Verify that regular expressions are free from elements causing exponential backtracking, and ensure untrusted input is sanitized to mitigate ReDoS or Runaway Regex attacks. | ✓ | ✓ | ✓ | 1333 |
 | **5.2.11** | [ADDED] Verify that the application appropriately sanitizes untrusted input before use in Java Naming and Directory Interface (JNDI) queries and that JNDI is configured as securely as possible to prevent JNDI injection attacks. | ✓ | ✓ | ✓ | 917 |
+| **5.2.12** | [ADDED] Verify that the application sanitizes content before it is being sent to memcache to prevent injection attacks. | | ✓ | ✓ | |
```

Comment:
"being" is not needed, based on grammarly

Reply:
Fair, I have fixed that.